eng/PatchConfig.props

```diff
@@ -28,6 +28,7 @@ Later on, this will be checked using this condition:
   </PropertyGroup>
   <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.11' ">
     <PackagesInPatch>
+      Microsoft.AspNetCore.Mvc.Core;
```

Comment:
@natemcmaster \ @dougbu does this look correct?

Reply:
Yes.

Reply:
Based on source code changes, looks like you need to update Mvc.RazorPages as well.